src/wp-includes/script-loader.php

```diff
@@ -659,6 +659,7 @@ function wp_default_packages_inline_scripts( &$scripts ) {
 					( wp_installing() && ! is_multisite() ) ? '' : wp_create_nonce( 'wp_rest' )
 				),
 				'wp.apiFetch.use( wp.apiFetch.nonceMiddleware );',
+				'wp.apiFetch.use( wp.apiFetch.mediaUploadMiddleware );',
 				sprintf(
 					'wp.apiFetch.nonceEndpoint = "%s";',
 					admin_url( 'admin-ajax.php?action=rest-nonce' )
```

Comment:
I think we need to update the versions of the scripts to match the updated package in this file.